Vizzuality/half-earth-v3
# Protection card

## Protection card update
### Description
This PR updates the Protection card on the local scene sidebar. 
![image](https://user-images.githubusercontent.com/6906348/88638674-1ac88780-d0bc-11ea-8a7c-183e64d1ddda.png)

The [merged WDPA layer](https://eowilson.maps.arcgis.com/home/item.html?id=0083c9adc3b548e78a9dfbaa4c909812) has been added for testing purposes on the scene settings, on the next iteration of this task this layer should be only activated on the local scene.

Also the African country flags have been added to test how the flags could be dinamically added depending on the selected country ISO code. Once this approach is approved the rest of the flags should be provided by the design team (there's already an ongoing [task for this purpose](https://half-earth-map.atlassian.net/browse/HE-32))
The flags files have been added to the `public` folder, find documentation about it on the [React docs](https://create-react-app.dev/docs/using-the-public-folder/). 
### Designs
The updated styles could be found on [the InVision prototype](https://projects.invisionapp.com/d/main#/console/17484384/421887229/preview)
### Testing instructions
From the data globe select a country and check the sidebar (in order to see some flag visit an African country)
### Feature relevant tickets
https://half-earth-map.atlassian.net/browse/HE-48
https://half-earth-map.atlassian.net/browse/HE-41
https://half-earth-map.atlassian.net/browse/HE-27
